scripts/build.js

```diff
@@ -0,0 +1,58 @@
+import fs from "fs";
+import replaceInFiles from "replace-in-files";
+
+const transpiledPathPrefix = "./build/src";
+
+async function build() {
+  await replaceInFiles({
+    files: [`${transpiledPathPrefix}/**/*.jsx`],
+    from: /export\s+default\s+function[^(]*\((.*)/gms,
+    to: (_match, rest) =>
+      `function MainComponent(${rest}\nreturn MainComponent(props, context);`,
+  });
```

Comment:
MainComponent should be the name of the component, try something like:

```javascript
  await replaceInFiles({
    files: [`${transpiledPathPrefix}/**/*.jsx`],
    from: /export\s+default\s+function\s+(\w+)\((.*)/gms,
    to: (_match, funcName, rest) =>
      `function ${funcName}(${rest}\nreturn ${funcName}(props, context);`,
  });
  ```

Reply:
That works works! No more white space and a proper function name
<img width="451" alt="Screenshot 2023-11-20 at 11 09 49" src="https://github.com/near/neardevhub-widgets/assets/28901891/76104764-116a-464f-9a0e-955196516141">